### PR TITLE
Create doc for mod_caps

### PR DESCRIPTION
**Pull request by kanes115**
This PR creates docs for mod_caps

### CODE REVIEW

File: doc/modules/mod_caps.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Entity Capabilities Module
+
+## Purpose
+This module provides a presence-based mechanism 
+for exchanging information about entity capabilities
+as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html).
+
+## Configuration
+This module expects two optional arguments that applies to [cache tab](https://github.com/processone/cache_tab):
```
Comment (goddammit) [inline doc/modules/mod_caps.md]: apply

**Comment by Nyco** [comment]
What is the cache used for? Does the server reply to the requesting client on behalf of the requested client? That would be nice to explain in this doc.

File: doc/modules/mod_caps.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Entity Capabilities Module
+
+## Purpose
+This module provides a presence-based mechanism 
+for exchanging information about entity capabilities
```
Comment (fenek) [inline doc/modules/mod_caps.md]: Too many line breaks. One sentence per line.

File: doc/modules/mod_caps.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
 # Entity Capabilities Module
 
 ## Purpose
-This module provides a presence-based mechanism 
-for exchanging information about entity capabilities
-as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html).
-It requests and caches Entity Capabilities from clients.
+This module provides a presence-based mechanism for exchanging information about entity capabilities as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html). Additionally, it filters out PEP messages that recipient declared (in announced caps) being not capable of handling.
```
Comment (goddammit) [inline doc/modules/mod_caps.md]: This module provides a presence-based mechanism for exchanging information about entity capabilities as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html). 
Additionally, it filters out PEP messages that the recipient declared (in announced caps) not being able to handle.
?

File: doc/modules/mod_caps.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
 # Entity Capabilities Module
 
 ## Purpose
-This module provides a presence-based mechanism 
-for exchanging information about entity capabilities
-as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html).
-It requests and caches Entity Capabilities from clients.
+This module provides a presence-based mechanism for exchanging information about entity capabilities as defined in [XEP-0115](https://xmpp.org/extensions/xep-0115.html). Additionally, it filters out PEP messages that recipient declared (in announced caps) being not capable of handling.  
+It is not this module's responsibility to intercept and answer to disco requests routed between clients.
```
Comment (goddammit) [inline doc/modules/mod_caps.md]: It is not this module's responsibility to intercept and answer disco requests routed between clients.

**Comment by kanes115** [comment]
@fenek, @goddammit, corrections made!

**Pull request merged**